backend/app/api/ai_agent.py

```
@@ -0,0 +1,281 @@
+
+
+# ── 보조: 최근 N 시간 그래프 (Design §5 의 선택적 용도) ────────────────────────
+
+
+@router.get(
+    "/activity/hourly",
+    dependencies=[Depends(get_current_user)],
+)
+async def get_activity_hourly(
+    hours: int = Query(default=24, ge=1, le=168),
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)